app/jobs/update_admin_claim_tasks_with_dqt_api_job.rb

```diff
@@ -0,0 +1,13 @@
+class UpdateAdminClaimTasksWithDqtApiJob < ApplicationJob
+  def perform(claim)
+    AutomatedChecks::UpdateAdminClaimTasksWithDqt.new(
```

Comment:
Fairly happy with the name.

Considered `From`, `With`, and `Via`. Preferred the last two because it collaborates with rather than the updates are directly from DQT. Thought about leaving out `WithDqt` altogether because it's describing an arg and tying it to a particular implementation but that's actually what it's doing for now so decided to leave in.

Deliberately left out generic descriptors like `data`. If the logic's applied across the name you get things like `UpdateAdminClaimTasksDataWithDqtData`. Should be cautious of using generic terms like `data`, `response`, `string`, etc, to describe things.